packages/pages/src/dev/server/public/templateBase.html

```diff
@@ -26,16 +26,22 @@
             throw new Error("Function not implemented.");
           },
         };
-        const { default: Component } = await template.getComponent();
+        const { default: Component, render } = await template.getComponent();
```

Comment:
getComponent renders the template module. Why not just have it return the default export if it exists and throw an error otherwise?

You shouldn't call get commit unless you're confident the template has a component.